packages/voila/src/manager.ts

```diff
@@ -7,93 +7,47 @@
  * The full license is in the file LICENSE, distributed with this software. *
  ****************************************************************************/
 
-import {
-  WidgetManager as JupyterLabManager,
-  WidgetRenderer
-} from '@jupyter-widgets/jupyterlab-manager';
+import { WidgetRenderer, output } from '@jupyter-widgets/jupyterlab-manager';
 
-import { output } from '@jupyter-widgets/jupyterlab-manager';
+import { KernelWidgetManager } from '@jupyter-widgets/jupyterlab-manager/lib/manager';
```

Comment:
Considering exporting this from the upstream package in https://github.com/jupyter-widgets/ipywidgets/pull/3166

Reply:
https://github.com/jupyter-widgets/ipywidgets/pull/3166 has been merged and released so we should be able to update the import.